src/routes/docs/products/auth/team-invites/+page.markdoc

```diff
@@ -0,0 +1,391 @@
+---
+layout: article
+title: Team invites
+description: Learn how to manage team invites in Appwrite. Implement both client-side email invites and server-side custom flows for team memberships.
+---
+
+Appwrite provides two approaches for adding members to teams: client-side email invites and server-side custom flows. Each approach serves different use cases and offers unique benefits.
+
+# Client-side email invites {% #client-side %}
```

Comment:
please remember the style guidelines on [headings](https://github.com/appwrite/website/edit/main/STYLE.md#headings).

Reply:
@stnguyen90 Should I change this to "Invite client-side"? Or "Invite client-side by email"?